DiscordRichPresence/SettingsFile.cpp

```diff
@@ -11,6 +11,22 @@ std::string GetExecutableFileName()
 	return std::string(buffer);
 }
 
+//used to convert strings to a usable string in MessageBox, example:
+//std::wstring stemp = s2ws(executablePath);
+//MessageBox(0, stemp.c_str(), L"", MB_OK);
+//https://stackoverflow.com/a/27296
+std::wstring s2ws(const std::string& s)
```

Comment:
Unused?

Reply:
Sorry, I need to put that into a personal dev scratchpad.  I used it to show the folder paths as my `settings.ini` can't be updated by the plugin and I was diagnosing that (and ran into the evil/fun aspects of C++ and strings).  Unable to solve my setting file save issues though... :(

Reply:
Ah, gotcha. No worries